README.md

```diff
@@ -1,12 +1,6 @@
 # liftwing-python
+<h1>LiftWing-python</h1>
+
+This is a Python module that *sends a user's request to the LiftWing API and gets back a response*.
+
```

Comment:
Just a suggestion. A module may be a single file containing python code while a package is a collection of modules.
```suggestion
# LiftWing python package

This is a Python package that acts as a client and allows users to make requests to the LiftWing API.
Its purpose is to allow users to interact with the API by writing python code instead of manipulating http requests.
```